src/js/login/containers/Main.jsx

```diff
@@ -49,15 +52,15 @@ class Main extends React.Component {
   }
 
   getLogoutUrl() {
-    $.ajax({
-      url: `${environment.API_URL}/v0/sessions`,
-      type: 'DELETE',
-      headers: {
+    fetch(`${environment.API_URL}/v0/sessions`, {
+      method: 'DELETE',
+      headers: new Headers({
         Authorization: `Token token=${sessionStorage.userToken}`
-      },
-      success: (result) => {
-        this.setState({ logoutUrl: result.logout_via_get });
-      }
+      })
+    }).then(response => {
```

Comment:
So this is a larger UX question that I asked when we were first building login. We ended up kicking the can on this in an effort to just get an MVP up on vets.gov. 

I think we need to bring @AnitraAppa into this convo as she has given much of our direction on error handling from a UX perspective which has allowed me to implement it from a technical perspective.